R/preprocess.R

```diff
@@ -34,7 +34,7 @@ enw_add_metaobs_features <- function(metaobs, holidays = c(),
   metaobs <- data.table::copy(metaobs)
   metaobs[, day_of_week := weekdays(date)]
 
-  # make holidays be sundays
+  # make holidays be Sundays
```

Comment:
Rstudio was complaining, I am not a that much of a grammar nazi just by myself :D